.buildkite/pipeline.yml

```diff
@@ -8,8 +8,8 @@ steps:
       nix-shell shell-linear-types.nix --command ''
     timeout: 120
     key: "install-linear-ghc"
-  - label: "Run tests (linear-types + Stack)"
+  - label: "Run tests (linear-types)"
     command: |
-        nix-shell --pure --run 'stack --nix --stack-yaml stack-linear.yaml build --pedantic --test jni jvm inline-java'
+        nix-shell --pure --run 'bazel test --test_output=all //jvm/tests/... //jni/tests/... //tests/... --config ghc_9_0_1' shell-linear-types.nix
```

Comment:
If listing the targets like this becomes unwieldy you could use [`--build_tag_filters`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--build_tag_filters) and [`--test_tag_filters`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_tag_filters) flags instead. But, listing them like this works too.

Reply:
Done.